data/meson.build

```diff
@@ -29,3 +29,5 @@ rauc_dbus_policy = configure_file(
   copy : true
 )
 install_data(rauc_dbus_policy, install_dir : dbuspolicydir)
+
+install_data('rauc.catalog', install_dir : systemdcatalogdir)
```

Comment:
What will happen if `systemdcatalogdir` is empty, due to meson not finding systemd?

Reply:
Changed to use `catalogdir` pkconfig variable and default to `libdir / 'systemd' / 'catalog'` so it cannot be empty anymore.